src/__init__.py

```diff
@@ -117,16 +117,35 @@ def create_items(self):
 
             if item_count == 0: continue
 
-            for i in range(item_count):
+            for _ in range(item_count):
                 new_item = self.create_item(name)
                 pool.append(new_item)
 
-            if item.get("early"): # only early
-                self.multiworld.early_items[self.player][name] = item_count
-            if item.get("local"): # only local
+            if item.get("early"): # Some or all early
+                if isinstance(item["early"],int) or (isinstance(item["early"],str) and item["early"].isnumeric()):
+                    self.multiworld.early_items[self.player][name] = int(item.get("early"))
```

Comment:
Seems redundant to check if early is an int then use `get` instead of just referencing the item directly. Is there something I'm missing?

Reply:
nope thats a mistake, Im leaving the int() so it convert if its a valid string but the `get` is a leftover from last version of this code